_sass/_customstyles.scss

```diff
@@ -334,6 +334,10 @@ img.screenshotSmall {
     max-width: 300px;
 }
 
+img.catalog {
```

Comment:
I wouldn't recommend taking this step yet - it's possible no new styles will be needed.

In general, if we have a set of practices and criteria for reviewing CSS, our approach on /website should be surfaced and described in Contributing.